src/Symfony/Component/PropertyInfo/Tests/TypeTest.php

```diff
@@ -85,4 +89,34 @@ public function testInvalidCollectionValueArgument()
 
         new Type('array', false, null, true, [new \stdClass()], [new Type('string')]);
     }
+
+    /**
+     * @dataProvider createProperInternalTypeDataProvider
+     *
+     * @param list<TypeInfoType> $types
+     * @param list<Type>         $legacyTypes
+     */
+    public function testCreateProperInternalTypes(array $types, array $legacyTypes)
+    {
+        $this->assertEquals($types, array_map(fn (Type $t): TypeInfoType => $t->internalType, $legacyTypes));
```

Comment:
```suggestion
        $this->assertEquals($types, array_column($legacyTypes, 'internalType'));
```

Reply:
not resolved it seems

Reply:
Indeed, I missed that part, thanks for poiting out :slightly_smiling_face: